src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -312,19 +321,24 @@ object DistributedUtil extends Logging {
     skipEmpty: Boolean,
     function: (Pileup, Pileup) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = reads1.context.broadcast(reference)
```

Comment:
Yeah, even though we don't currently map over pileups more than once in any callers, I think that is a case that will come up eventually and we should not make it require broadcasting the reference genome each time. Since these pileupFlatmap functions are already taking RDD's and thus are already using Spark infra, maybe best to just have them take a`Broadcast[ReferenceGenome]`?